### PR TITLE
grammar/wording updates

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: ffv1.md [hunk 1]
```diff
@@ -1111,7 +1111,7 @@ If state_transition_delta is not present in the FFV1 bitstream, all Range coder
 | Other | reserved for future use         | reserved for future use         | reserved for future use         |
 
 Restrictions:  
-If `colorspace_type` is 1, `chroma_planes` MUST be 1, `h_chroma_subsample` MUST be 1, `v_chroma_subsample` MUST be 1.  
+If `colorspace_type` is 1, then `chroma_planes` MUST be 1, `h_chroma_subsample` MUST be 1, and `v_chroma_subsample` MUST be 1.
```
Comment (JeromeMartinez) [inline ffv1.md]: I missed this one in the patch about chroma_planes, but should actually be `log2_h_chroma_subsample` and `log2_v_chroma_subsample`

Comment (dericed) [reply ffv1.md]: not `log2(h_chroma_subsample)`?

Comment (JeromeMartinez) [reply ffv1.md]: no, we changed it.
check `Parameters( )` pseudo-code `log2_h_chroma_subsample` definition part

Comment (dericed) [reply ffv1.md]: fixed in a second commit, thx for catching this.

**Comment by michaelni** [comment]
+If `colorspace_type` is 1, then `chroma_planes` MUST be 1, `log2_h_chroma_subsample` MUST be 1, and `log2_v_chroma_subsample` MUST be 1.

shouldnt this be 0 ?

**Comment by JeromeMartinez** [comment]
> shouldnt this be 0 ?

I confirm, typo during move to `log2`, it should be 0.

**Comment by dericed** [comment]
Updated to 0, please re-review.

**Comment by JeromeMartinez** [comment]
bump on this, is there something blocking? IMO it is good to have such correction from a native English speaker.

**Pull request closed without merging**